3-box-model/style.css

```diff
@@ -0,0 +1,26 @@
+header {
+    background-color: black;
+    border: 2px solid green;
+    color: white;
+    position: sticky;
+}
+
+nav {
+    border: 2px solid blue;    
+    box-sizing: border-box;
+    position: absolute;
+    width: 200px;
```

Comment:
for the exercise you need to make the element as big as 200px but without using the width property.